### ISSUE TITLE
Connecting to dedi server with VerifySignatures enabled kicks 

### ISSUE THREAD

**Issue by hobart666** [Fullerpj/United-States-Air-Force-Mod#91]
Hi Sir,
 I'm trying to run this on a dedicated server but when I connect I get kicked immediately and see this in the server console Wrong signature for file @usaf\addons\ldl_ac130.pbo
I'm kinda a noob with arma editing but what can I do to make this right and keep verifying the siggies?

 Thank you Sir,
 hobart

**Comment by Fullerpj** [comment]
I posted the signature in the thread. It actually should be on the main
page.

**Comment by hobart666** [comment]
Howdy Sgt. Fuller, I'm Josh.
The signature for the keys folder is there I do have that in my server's
keys folder it's the one that you have linked for the ac130 that people
were asking about on the bia forums a while back. I got that one from the
Dropbox link and replaced the one in the God's add-ons folder and alas I am
still having to disable signature checks (I did make sure that the one on
my client comp was the one from Dropbox as well.)
Any help would be cool, if you're in the world of hurry up & wait still
that's cool I'm patient. Great mod by the way!
